src/main/kotlin/fi/metatavu/lipsanen/projects/milestones/tasks/TaskAssigneeEntity.kt

```diff
@@ -0,0 +1,19 @@
+package fi.metatavu.lipsanen.projects.milestones.tasks
+
+import jakarta.persistence.*
+import java.util.*
+
+@Entity
+@Table(name = "task_assignee")
+class TaskAssigneeEntity {
+
+    @Id
+    lateinit var id: UUID
+
+    @ManyToOne
+    @JoinColumn(name = "task_id", nullable = false)
+    lateinit var taskId: UUID
```

Comment:
```suggestion
```
This might be not needed at all since you manage the relation from the parent's (TaskEntity side)